utils/utils.tsx

```diff
@@ -958,32 +962,32 @@ export function getPxToSubstract(char = '@') {
     return 0;
 }
 
-export function setSelectionRange(input, selectionStart, selectionEnd) {
+export function setSelectionRange(input: HTMLInputElement, selectionStart: number, selectionEnd: number) {
     if (input.setSelectionRange) {
         input.focus();
         input.setSelectionRange(selectionStart, selectionEnd);
-    } else if (input.createTextRange) {
-        var range = input.createTextRange();
+    } else if ((input as any).createTextRange) {
```

Comment:
Similarly, `setSelectionRange` is also supported everywhere, so we can get rid of this block too